_includes/use-cases/use-case-transport-download-file.md

```diff
@@ -0,0 +1,3 @@
+Download file that corresponds to the given id.
```

Comment:
Download the File's binary content that corresponds to the given `fileId`

Reply:
Resolved in other branch since the format in this one is also wrong (no Parameters etc.)